src/inet/http_server.c

```diff
@@ -138,7 +145,7 @@ static int read_request(struct http_server_t *self_p,
                         struct http_server_request_t *request_p)
 {
     int res;
-    char buf[128];
```

Comment:
in some browser, user-agent header is very big.
some user use the browser visit device http server
and then device will bei crashed.....

may be we can ignore user-agent header, alse a better idear